src/main/java/uk/gov/hmcts/dm/controller/StoredDocumentController.java

```diff
@@ -211,13 +217,13 @@ public ResponseEntity<Void> getBinary(@PathVariable UUID documentId, HttpServlet
                 logger.debug("documentId : {} has Request Header {} = {}", documentId.toString(), key, value));
         }
         if (toggleConfiguration.isChunking()) {
```

Comment:
This flag is disabled on all environments including Prod.
https://github.com/hmcts/document-management-store-app/blob/master/charts/dm-store/values.yaml#L53
https://github.com/hmcts/cnp-flux-config/blob/master/apps/dm-store/dm-store/prod.yaml

Reply:
I ma de it info as it is disabled, just in case  if somewhere it is enabled